components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -378,6 +381,23 @@ export class ClusterServiceServer {
         }
     }
 
+    public async getScore(){
+        // TODO(arthur): findFiltered needs a proper filter.
+        // How can I filter with "The cluster where this component is running"?
+        for (const cluster of await this.clusterDB.findFiltered({})) {
+            return new Promise<number>((resolve) => {
+                const clusterStatus = convertToGRPC(cluster);
+                resolve(clusterStatus.getScore())
+            });
+        }
+        // This should never be called.
+        // If gitpod_cluster_score is set to -1, it means ws-manager-bridge failed to list
+        // it's own cluster.
+        return new Promise<number>((res) => {
+            res(-1)
+        });
```

Comment:
No errors, but I'm seeing this now:
![image](https://user-images.githubusercontent.com/24193764/132589176-ca1e371b-df53-4bdb-9e86-5b463863a023.png)


I'm assuming that happens because I haven't passed a proper filter to `clusterDB.findFiltered` function. 

How can I use the filter to find only the cluster where this ws-manager-bridge runs on?

Reply:
This code should be in `BridgeController` - there you'll also find examples how to filter the workspace cluster